runtime/tchannel_server.go

```diff
@@ -112,10 +112,13 @@ func NewTChannelEndpointWithPostResponseCB(
 		zap.String("handlerID", handlerID),
 		zap.String("method", method),
 	)
+
+	cleanMethod := strings.Replace(method, ":", "-", -1)
```

Comment:
Perhaps move this to util method, so it can be used by tchannel client and server.
In fact, I think tally already has a sanitize method.

Reply:
tally has a sanitize class thats more complicated then needed. A string replace is sufficient here.